Tests/CocoaPodsTest/PodInstall/Podfile.lock

```diff
@@ -67,4 +67,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: ce6227c2aeeca0d51c521648629d4f06ae38a439
 
-COCOAPODS: 1.11.2
```

Comment:
Downgrading the version might not be a good idea. Should we use bundler to fix the Cocoapods version?